vignettes/SingleR.Rmd

```diff
@@ -195,10 +195,11 @@ plotScoreHeatmap(pred, clusters=sceG$donor)
 # Available reference datasets
 
 `r Biocpkg("SingleR")` also directly provides a number of reference datasets generated from bulk RNA-seq of pure cell types.
+
 There are two data sets from human cells (Human Primary Cell Atlas and Blueprint/ENCODE) and two data sets from mouse cells (e.g. Immunological Genome Project). 
 For more details see the vignette dedicated to the `r Biocpkg("SingleR", vignette = "datasets.html", label="reference datasets")`.
 
-Each reference dataset is obtained with a specific function: `HumanPrimaryCellAtlasData`, `BlueprintEncodeData`, `ImmGenData`, `MouseBulkData`.
+Each reference dataset is obtained with a specific function: `HumanPrimaryCellAtlasData`, `BlueprintEncodeData`, `ImmGenData`, `MouseData`.
```

Comment:
I usually refer to functions with parentheses to make it very clear I'm talking about a function, and not an object/argument/class. So it would be `HumanPrimaryCellAtlasData()`.